loleaflet/src/core/Util.js

```diff
@@ -164,10 +164,6 @@ L.Util = {
 		if (!useExactDPR)
 			dpiScale = Math.ceil(dpiScale);
 
-		if (dpiScale == 1 && L.Browser.retina) {
```

Comment:
The removal here does not look like a cosmetic change, but a functional one?

Reply:
I thought that this code never gets hit. If L.Browser.retina  equals to "true", then dpiScale should not equal to 1.